api/tests/Feature/ApplicantTest.php

```diff
@@ -978,12 +978,14 @@ public function testCountApplicantsQuerySuspended(): void
         ]);
         PoolCandidate::factory()->count(5)->availableInSearch()->create([
             'pool_id' => $pool1,
+            'suspended_at' => null,
             'user_id' => User::factory([
                 'job_looking_status' => ApiEnums::USER_STATUS_ACTIVELY_LOOKING,
             ])
         ]);
         PoolCandidate::factory()->count(4)->suspended()->create([
             'pool_id' => $pool1,
+            'suspended_at' => config('constants.past_date'),
             'user_id' => User::factory([
                 'job_looking_status' => ApiEnums::USER_STATUS_ACTIVELY_LOOKING,
             ])
```

Comment:
Currently running the tests. 

That is odd though, it is calling `suspended()` in the chain which does set a `suspended_at`. 😕 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/736c3c31fa2dfea262ce57d8098f25b77cc2201b/api/database/factories/PoolCandidateFactory.php#L104-L112

Reply:
Hmm, I never even saw that. Could it be the `now` that was maybe causing issues? I see check `>= now` in the scope so maybe on the odd chance it generates the upper range they are equal and it is included? 🤔

Reply:
That seems like the most likely culprit in that case.

To rule out any issues with rounding we could generate two more pool candidates for the test. 
One with
`suspended_at = Carbon::now()->subMinute())`

and the other
`suspended_at = Carbon::now()->addMinute())`

The expected candidate count then would go from 5 to 6.

Reply:
So, I updated the `suspended` logic to just set it to one min earlier and that also seems to work after 50 re-runs 🤷‍♀️ 

[update suspended logic on pool candidate factory](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6415/commits/2e3fe614ddccb7fc8e54aa005a026ba3243edeb2)

Reply:
weird
guess the factory was generating "now" much more frequently than you'd expect

Reply:
I guess so 😬